src/templates/drf/form-field/form-field.html

```diff
@@ -1,5 +1,5 @@
 <div class="control-group" ng-class="{error: field.errors.length, required: field.required}">
-    <label ng-if="label" class="control-label">
+    <label ng-if="label" class="control-label" for="{{ scope.field.id }}">
```

Comment:
We defined our property as `scope.id` so you don't need `field` and also `scope` is assumed in the template so that shouldn't be there too. Just `{{ id }}`